index.html

```diff
@@ -1,12 +1,13 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <title>Most Awesome Landing Page You've Ever Seen</title>
-    <link rel="stylesheet" href="css/style.css">
-  </head>
-
-  <body>
+<html lang="zxx">
+<head>
+  <meta charset="UTF-8">
```

Comment:
Your filler text of Lorem itsum gives a good idea of what the final build of the website will be. For HTML best practices, please, however, use **lang** attribute **"en"** for normal English translated websites.